tests/js/tests/yii.captcha.test.js

```diff
@@ -0,0 +1,192 @@
+var assert = require('chai').assert;
+var sinon;
+var withData = require('leche').withData;
+var jsdom = require('mocha-jsdom');
+
+var fs = require('fs');
+var vm = require('vm');
+
+describe('yii.captcha', function () {
+    var yiiCaptchaPath = 'framework/assets/yii.captcha.js';
+    var jQueryPath = 'vendor/bower/jquery/dist/jquery.js';
```

Comment:
@dynasource I used jQuery that ships with Yii2 by default. This prevents from adding the same package to NPM (other packages include `punycode` for example in #12936) and synchronizing the versions on changes. I described it in #12936 and it was approved. With the current implementation of loading framework's JS dependencies I think it's better. When we get rid of Composer Asset Plugin (it will happen in 2.1 I believe) or make js less coupled with PHP, we can remove this as well.